src/System.IO.Compression/src/System/IO/Compression/DeflateManaged/InflaterManaged.cs

```diff
@@ -64,17 +64,20 @@ internal sealed class InflaterManaged
         private readonly byte[] _codeLengthTreeCodeLength;
         private readonly bool _deflate64;
         private HuffmanTree _codeLengthTree;
+        private long _uncompressedSize = -1;
+        private long _currentInflatedCount;
```

Comment:
It looks like `_uncompressedSize` should be readonly (and not initialized to anything) since it is only, unconditionally, assigned in the solitary constructor.